docs/onboarding_to_cluster.md

```diff
@@ -0,0 +1,153 @@
+# Onboarding to a cluster
+
+Currently we support 2 ways to request/perform onboarding to any supported clusters:
+
+1. Onboarding request - please use the **Onboarding to Cluster** issue template in [operate-first/support](https://github.com/operate-first/support) repository.
+2. Opening a PR with the desired changes. The rest of this doc focuses on this second option. We'd gladly welcome your contributions.
+
+## Prerequisites
+
+You'll require the following tools:
+
+- [kustomize](https://kustomize.io/)
+- [sops](https://github.com/mozilla/sops) and [ksops](https://github.com/viaduct-ai/kustomize-sops) - for encryption of sensitive data. If you want to hand us over any PII data like user emails, etc. you need to encrypt it
```

Comment:
Question: We also have a toolbox available on github right? Should we instead tell people to use that?

Reply:
Good point! We should offer it as the default option!